src/nidcpower/metadata/attributes.py

```diff
@@ -2305,11 +2305,14 @@
     },
     1150302: {
         'access': 'read-write',
-        'attribute_class': 'AttributeIsolationState',
         'documentation': {
             'description': '\nDefines whether the channel is isolated.\n',
             'note': '\nThis attribute is not supported on all devices. For more information about supported devices, search ni.com for Supported Attributes by Device.\n'
         },
+        'enum_value_to_bool_map': {
+            1128: True,
```

Comment:
This is a not a good way to define things, in my opinion.
The enum is presumably already in metadata. I'd like for the metadata _of the enum_ to specify which value corresponds to True and which value corresponds to False.

Let's break this up in several steps / PRs:

1. Add the attribute as an enum.
2. Mark the enum as private. This may or may not be a build error (it should be). The enum should start with `_` and not be included in documentation because it's private.
3. Add a way to specify, in enum metadata, what values map to.
5. In nimi-python world, codegen "converter" functions for those enums. The name of the converter function should be in "expanded" metadata.
6. Add a new class "AttributeWithEnumConverter" or something like that.
7. Modify nimi-python so that the attribute, if enum, uses the converter.

Reply:
This has been addressed through the latest changes.